nexus/src/external_api/http_entrypoints.rs

```diff
@@ -8884,6 +8890,66 @@ async fn role_view(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+// Current user
+
+/// Fetch the user associated with the current session
+#[endpoint {
+   method = GET,
+   path = "/v1/current-user",
+   tags = ["hidden"],
```

Comment:
why hidden? My understanding of the "hidden" tag originally was that we didn't want to actually hide it from the OpenAPI document for ... convenience? I think these interfaces really are first-class and supported, right?

Reply:
I was just carrying these over. I'll update it.